controller/users.go

```diff
@@ -632,6 +643,12 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 	return ctx.OK(ConvertToAppUser(ctx.RequestData, user, identity))
 }
 
+func (c *UsersController) sendVerificationEmail(ctx context.Context, user account.User) error {
```

Comment:
Any specific reason to define this method instead of just using c.EmailVerificationService.SendVerificationCode(ctx, user) directly where needed?

Reply:
Yeah it's a one liner, was expecting to do other things too :)  
removed e8b8863